src/Extracting/Strategies/BodyParameters/GetFromFormRequest.php

```diff
@@ -196,6 +196,7 @@ protected function normaliseRules(array $rules)
         $testData = [];
         foreach ($rules as $key => $ruleset) {
             if (!Str::contains($key, '.*')) continue;
+            if (!is_string($ruleset) && !($ruleset instanceof Rule)) continue;
```

Comment:
Did you test this? I think the `ruleset` here is actually the group of all rules, which should be an array. I believe this should go in `parseRule()`, the function that parses each rule individually. Even better, this method (where therre's already a check for `Rule` class):

https://github.com/knuckleswtf/scribe/blob/73351b170450d69e4039edd58034a883d4b0648e/src/Extracting/Strategies/BodyParameters/GetFromFormRequest.php#L399-L409